birdset/modules/models/linear_classifier.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch import nn
+from typing import Optional
+
+class LinearClassifier(nn.Module):
+    
+    def __init__(
+        self,
+        num_classes: int,
+        in_features: int   
+    ) -> None:
+        """
+        Initialize the LinearClassifier.
+        
+        Args:
+            num_classes (int): The number of output classes for the classifier.
+            in_features (int): The size of the input for the linear classifier.
+        """
+        super().__init__()
+        self.classifier = nn.Linear(in_features, num_classes)
```

Comment:
@XgamerTV This is now a 2 layer NN, why did you choose 2 layers instead of one?

Reply:
I thought it would be one layer from the embedding suze to the class amount how can I use one?